server/api/contact.post.ts

```diff
@@ -9,33 +76,36 @@ export default defineEventHandler(async (event) => {
     return sendRedirect(event, '/en/contact/failed/');
   }
 
-  return fetch(
-    'https://api.postmarkapp.com/email',
-    {
-      method: 'POST',
-      headers: {
-        'Accept': 'application/json',
-        'Content-Type': 'application/json',
-        'X-Postmark-Server-Token': config.postmarkApiToken,
-      },
-      body: JSON.stringify({
-        'MessageStream': 'contact-form',
-        'From': `"${body.name}" <forms@voorhoede.nl>`,
-        'To': 'post@voorhoede.nl',
-        'Subject': 'Voorhoede website contact form',
-        'TextBody': [
-          ['From:', body.email],
-          body.business && ['Company:', body.business],
-          body.phone && ['Phone:', body.phone],
-          ['Body:', body.explanation],
-        ]
-          .filter(Boolean)
-          .map((pair) => [...pair, '\n'])
-          .flat()
-          .join('\n'),
-      }),
+  handlePipedriveCommunication(
+    config.pipedriveApiToken,
+    config.pipedriveApiUrl,
+    body,
+  );
```

Comment:
intentionally no await here, so the user doesn't get slowed down by the pipedrive interaction.